app/controller.js

```diff
@@ -1,31 +1,35 @@
-import axios from 'axios';
-import { createSolutionPageContext } from './contextCreator/createSolutionPageContext';
-import { createSolutionListPageContext, createFoundationSolutionListPageContext } from './contextCreator/createSolutionListPageContext';
-
-export const getSolutionFoundationListPageContext = async () => {
-  const foundationSolutionListResponse = await axios.get('http://localhost:8080/api/v1/Solutions/Foundation');
-
-  const context = createFoundationSolutionListPageContext(
-    foundationSolutionListResponse.data.solutions,
-  );
-
-  return context;
-};
-
-export const getSolutionListPageContext = async () => {
-  const solutionListResponse = await axios.get('http://localhost:8080/api/v1/Solutions');
-
-  const context = createSolutionListPageContext(
-    solutionListResponse.data.solutions,
-  );
-
-  return context;
-};
-
-export const getSolutionPageContext = async (solutionId) => {
-  const solutionData = await axios.get(`http://localhost:8080/api/v1/Solution/${solutionId}`);
-
-  const context = createSolutionPageContext(solutionData.data.solution);
-
-  return context;
-};
+import axios from 'axios';
+import { createSolutionPageContext } from './contextCreator/createSolutionPageContext';
+import { createSolutionListPageContext, createFoundationSolutionListPageContext } from './contextCreator/createSolutionListPageContext';
+
+export const getSolutionFoundationListPageContext = async () => {
+  const foundationSolutionListResponse = await axios.get('http://localhost:8080/api/v1/Solutions/Foundation');
+
+  const context = createFoundationSolutionListPageContext(
+    foundationSolutionListResponse.data.solutions,
+  );
+
+  return context;
+};
+
+export const getSolutionListPageContext = async () => {
+  const solutionListResponse = await axios.get('http://localhost:8080/api/v1/Solutions');
+
+  const context = createSolutionListPageContext(
+    solutionListResponse.data.solutions,
+  );
+
+  return context;
+};
+
+export const getSolutionPageContext = async (solutionId) => {
+  const solutionData = await axios.get(`http://localhost:8080/api/v1/Solution/${solutionId}`);
+
+  const context = createSolutionPageContext(solutionData.data.solution);
+
+  return context;
+};
+
```

Comment:
I removed the getSolutionPageContext. Are we still using this function?

Reply:
probably not I will go through and delete the unused ones